cp-ksql-cli/Dockerfile

```diff
@@ -0,0 +1,34 @@
+ARG DOCKER_UPSTREAM_REGISTRY
+ARG DOCKER_UPSTREAM_TAG
+
+FROM ${DOCKER_UPSTREAM_REGISTRY}confluentinc/cp-base:${DOCKER_UPSTREAM_TAG}
+
+EXPOSE 8088
+
+ENV COMPONENT=ksql-cli
+ENV KSQL_CLASSPATH=/usr/share/java/${COMPONENT}/*
+
+ARG COMMIT_ID=unknown
+LABEL io.confluent.docker.git.id=$COMMIT_ID
+ARG BUILD_NUMBER=-1
+LABEL io.confluent.docker.build.number=$BUILD_NUMBER
+LABEL io.confluent.docker=true
+
+ARG PROJECT_VERSION
+ARG ARTIFACT_ID
+
+ADD target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/java/${ARTIFACT_ID}/* /usr/share/java/${COMPONENT}/
+ADD target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/share/doc/* /usr/share/doc/${ARTIFACT_ID}/
+ADD target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/bin/* /usr/bin/
+ADD target/${ARTIFACT_ID}-${PROJECT_VERSION}-package/etc/* /etc/ksql/
+
+COPY include/etc/confluent/docker /etc/confluent/docker
+
+RUN mkdir -p /etc/${COMPONENT} /etc/${COMPONENT}/secrets /var/log/${COMPONENT}
+
+RUN bash /etc/confluent/docker/configure
+ENV KSQL_LOG4J_OPTS=-Dlog4j.configuration=file:/etc/${COMPONENT}/log4j.properties
+
+ENTRYPOINT ["ksql"]
+CMD ["http://localhost:8088"]
```

Comment:
This should probably be a real command -- what should the image do when called with no args?

Reply:
What do you mean by real command? The `ksql` cli command without args will try to connect to the server at `http://localhost:8088`. So technically this extra command is totally redundant.

Not sure if we have too many more options here. The only other option is to not provide any start command, so the user would explicitly have to do `docker run -it -rm confluentinc/cp-ksql-cli ksql <server address>`. 

I think omitting the extra `ksql` command is a bit better, but am open to suggestions.

Reply:
Ah ha.. I didn't realize the behavior of CMD when used with ENTRYPOINT -- I withdraw my comment.

Reply:
Perhaps @gnschenker knows the pros/cons here?